doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -3262,7 +3262,7 @@ NCSDK-15512: Modem traces retrieval incompatible with TF-M
 Libraries for networking
 ========================
 
-.. rst-class:: v2-9-0-nRF54H20-1 v2-9-1 v2-9-0 v2-8-0
+.. rst-class:: v3-0-0 v2-9-0-nRF54H20-1 v2-9-1 v2-9-0 v2-8-0
 
 CIA-1400: :ref:`lib_nrf_cloud` FOTA and P-GPS downloads over CoAP might fail to resume
```

Comment:
@maxd-nordic Do you know if this one is fixed?

https://nordicsemi.atlassian.net/browse/CIA-1400